### ISSUE TITLE
cockpit-composer on Fedora 34 crashes immediately

### ISSUE THREAD

**Issue by ondrejbudai** [osbuild/cockpit-composer#1303]
### Description of problem
cockpit-composer on Fedora 34 crashes with `The iterator does not provide a 'throw' method`

I tried the same on Fedora 33 with `cockpit-composer-30-1.fc33.noarch` and I cannot it reproduce it there.

### Version or commit hash (if applicable)
`cockpit-composer-30-1.fc34.noarch`

### How often reproducible
Always

### Steps to Reproduce

1. install the newest osbuild-composer from testing: `sudo dnf upgrade --enablerepo=updates-testing --advisory=FEDORA-2021-5a4b58555d`
2. restart osbuild-composer: `sudo systemctl restart osbuild-composer`
3. open http://localhost:9090/composer

### Actual results
![image](https://user-images.githubusercontent.com/119861/122195615-d04c2700-ce96-11eb-892b-32c3f9c9390c.png)

### Additional info
Log:
```
errorloadBlueprintsSaga TypeError: The iterator does not provide a 'throw' method
    w runtime.js:349
    _invoke runtime.js:267
    g runtime.js:118
    Redux 44
    fetchingBlueprints index.js:219
    componentDidMount index.js:58
    React 2
    unstable_runWithPriority scheduler.production.min.js:19
    React 9
    ar main.js:55
    promise callback*rr main.js:74
    <anonymous> main.js:81
    Webpack 3
blueprints.js:44:12
```

**Comment by jkozol** [comment]
@ondrejbudai I am unable to replicate this. I tried on my existing f34 machine and on a fresh one.

1) I launched this image: https://download.fedoraproject.org/pub/fedora/linux/releases/34/Cloud/x86_64/images/Fedora-Cloud-Base-34-1.2.x86_64.qcow2
2) I ran `dnf install cockpit-composer`
3) I updated osbuild-composer `sudo dnf upgrade --enablerepo=updates-testing --advisory=FEDORA-2021-5a4b58555d`
4) I started the osbuild-composer weldr socket through the UI and it launched successfully.
5) I restarted the service with `sudo systemctl restart osbuild-composer` and it still loaded as expected.
